build.py

```diff
@@ -16,11 +16,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Usage: build.py <0 or more of accessible, core, generators, langfiles>
+# Usage: build.py <0 or more of core, generators, langfiles>
 # build.py with no parameters builds all files.
 # core builds blockly_compressed, blockly_uncompressed, and blocks_compressed.
-# accessible builds blockly_accessible_compressed,
-#  blockly_accessible_uncompressed, and blocks_compressed.
 # generators builds every <language>_compressed.js.
 # langfiles builds every msg/js/<LANG>.js file.
 
```

Comment:
Should the below say the script generates two versions? And remove the "first pair" part?

Reply:
Done.